src/doom/g_game.c

```diff
@@ -972,7 +972,31 @@ static void SetMouseButtons(unsigned int buttons_mask)
 // Get info needed to make ticcmd_ts for the players.
 // 
 boolean G_Responder (event_t* ev) 
-{ 
```

Comment:
Please keep the whitespace in line you don't modify.

Reply:
Sorry, I just discovered I had "all whitespace changes ignored" on in TortoiseMerge. Corrected in 9430f1e, as well as another instance on line 1209.